src/zodbupdate/serialize.py

```diff
@@ -160,8 +160,8 @@ def __update_symb(self, symb_info):
         """
         if symb_info in SKIP_SYMBS:
             self.__skipped = True
-            return symb_info
-        elif symb_info in self.__renames:
+
+        if symb_info in self.__renames:
```

Comment:
I had to change this to not return so that references will still be updated to use bytes.

Reply:
Yes it makes sense, since we do not care about skipping for the data dict.